examples/01_tutorials/plot_first_level_model_details.py

```diff
@@ -0,0 +1,352 @@
+"""Studying firts-level-model details in a trials-and-error fashion
+================================================================
+
+In this tutorial, we study the parametrization of the first-level
```

Comment:
Could you explicitly specify atleast one parametrization method you are tying to trial-and-error ?

Reply:
Or in other words the concepts you are explaining here ?